report-hazard/index.html

```diff
@@ -0,0 +1,134 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <script src="report-hazard.js" defer></script>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <script src="https://unpkg.com/maplibre-gl@2.4.0/dist/maplibre-gl.js"></script>
+	<link href="https://unpkg.com/maplibre-gl@2.4.0/dist/maplibre-gl.css" rel="stylesheet" />
+    <link rel="stylesheet" href="style.css" />
+    <!-- <script type="module" src="script.js"></script> -->
+    <title>Document</title>
+</head>
+
+<body>
+    <header>
+        <h1>CampBuddy</h1>
+    </header>
+    <div class="toolbar">
+        <!-- header area shared among all pages. It also include navigation buttons -->
+        <a title="Select the location" href="#step1">1</a>
+        <span>-----</span>
+        <a title="What type of hazard would you like to report?" href="#step2">2</a>
+        <span>-----</span>
+        <a title="What type of wild animal did you spot?" href="#step3">3</a>
+        <span>-----</span>
+        <a title="Would you like to mention any further details?" href="#step4">4</a>
+        <span>-----</span>
+        <a title="Please upload photos to help others see the reported hazard visually." href="#step5">5</a>
+        <span>-----</span>
+        <a title="Review the report before submitting." href="#step6">6</a>
+        <span>-----</span>
+        <a href="#step7">7</a>
+    </div>
+    <main>
+        <form id="form">
+            <div class="page" id="step1">
+                <h1>1. Select the location.</h1>
+                <h1 id="loc"></h1>
+                <div id="map"></div>
+                <label>Your current location is:</label>
+                <input type="text" id="name" required>
+                <br>
+                <a href="#step2">Continue</a>
+            </div>
+            <div class="page" id="step2">
+                <h1>2. What type of hazard would you like to report?</h1>
+                <input type="radio" name="category" value="Wildlife">Wildlife<br>
+                <input type="radio" name="category" value="Weather">Weather<br>
+                <input type="radio" name="category" value="Infrastructure">Infrastructure<br>
+                <input type="radio" name="category" value="Campsite">Campsite<br>
+                <a href="#step1">Back</a>
+                <a href="#step3">Continue</a>
+            </div>
+            <div class="page" id="step3">
+                <h1>3. What type of wild animal did you spot?</h1>
+                <input type="radio" name="type" value="Bear">Bear<br>
```

Comment:
remove all <br>, even if it doesn't look good, we can add the styles later

Reply:
commit a55624b86862fa23dbbbd28c4baffa52e78877d1

Reply:
commit a93d2cf3df51cfdc8cc03299223ad0172d0e963e